spec/amp-var-substitutions.md

```diff
@@ -32,422 +29,868 @@ will still update to the most current values of the variables.  Note that some
 variables will not allow for expansion for A4A creatives.  Specifics will be
 addressed at a later time.
 
-## Page and Content
+## Variables 
+
+The tables below list the available URL variables grouped by type of usage. Further down in this document, are [descriptions](#variable-descriptions) of each of the variables, along with example usages.
+
+* [Page and Content](#page-and-content)
```

Comment:
my thinking was along the lines that this results in 2 levels of tabels to get to the variables. Given that this is a small table and the links from here go directly below, maybe this is not needed.

But, I am okay with whatever you decide here and will go with that.